include/xtensor/xmasked_view.hpp

```diff
@@ -160,6 +160,8 @@ namespace xt
         template <class D, class M>
         xmasked_view(D&& data, M&& mask);
 
+        xmasked_view(xmasked_view const&) = default;
```

Comment:
Why can't we rely on the compiler to generate it?

Reply:
```
error: definition of implicit copy constructor for 'xmasked_view<xt::xarray_container<xt::uvector<double>, xt::layout_type::row_major, xt::svector<unsigned long, 4, std::allocator<unsigned long>, true>> &, xt::xarray_container<xt::uvector<bool>, xt::layout_type::row_major, xt::svector<unsigned long, 4, std::allocator<unsigned long>, true>> &>' is deprecated because it has a user-declared copy assignment operator [-Werror,-Wdeprecated-copy]
```

Reply:
Oh, I totally missed this operator at the end of the class. Thanks!